projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item/cart-item.component.spec.ts

```diff
@@ -205,4 +206,93 @@ describe('CartItemComponent', () => {
       );
     });
   });
+
+  describe('Depicting configurable products in the cart', () => {
```

Comment:
Add tests for status summary containing successes (see above)

Reply:
will do

Reply:
done